identity/src/commonMain/kotlin/com/android/identity/securearea/config/SecureAreaConfiguration.kt

```diff
@@ -0,0 +1,19 @@
+package com.android.identity.securearea.config
+
+import com.android.identity.cbor.annotation.CborSerializable
+import com.android.identity.crypto.EcCurve
+import com.android.identity.securearea.KeyPurpose
+import com.android.identity.securearea.SecureArea
+
+/**
+ * Configuration for a specific secure area [SecureArea] to use.
+ */
+@CborSerializable
+sealed class SecureAreaConfiguration(
```

Comment:
Does this limit us to `SecureArea`s defined in this library? I guess only if you want to use the `applyConfiguration()` methods, right?

Reply:
With current implementation, it does limit us, but we can always define an open-ended variant that would have some basic fields + `Map<String, DataItem>` as an extension point.